contracts/meta-oracles/proxies/LegacyMakerOracleProxy.sol

```diff
@@ -0,0 +1,99 @@
+/*
+    Copyright 2020 Set Labs Inc.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+*/
+
+pragma solidity 0.5.7;
+
+import { Authorizable } from "../../external/SetProtocolContracts/lib/Authorizable.sol";
+
+import { IOracle } from "../interfaces/IOracle.sol";
+
+
+/**
+ * @title LegacyMakerOracleProxy
+ * @author Set Protocol
+ *
+ * Permissioned oracle proxy that is intended to replicate the interface of the MakerDAO Legacy IMedian
+ * oracle that returns bytes32 values.
+ */
+contract LegacyMakerOracleProxy is
```

Comment:
At this point it's not touching Maker's oracles so this name feels like a bit of a misnomer. This contract is more to convert our OracleProxies to interact with our legacy contracts.

Reply:
Maybe `LegacyContractOracleAdapter` or something like that is better

Reply:
Maybe `MedianOracleAdapter` since we're trying to conform to `IMedian`